src/xedgewarp.c

```diff
@@ -1,19 +1,36 @@
 // vim:ts=4:sw=4:expandtab
 #include "all.h"
 #include <getopt.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/wait.h>
 
 #ifndef __VERSION
 #define __VERSION "unknown"
 #endif
 
+typedef void (*callback)(void);
+
+/* Forward declarations */
+static void run();
+static void safe_fork(callback child_callback);
+
 xcb_connection_t *connection;
 xcb_window_t root;
-Config config;
+Config config = { .fork_mode = false };
```

Comment:
This is technically not necessary since it's the default value. I guess being explicit doesn't hurt, though the other values rely on the default, too. I'll add the other values here myself.